pkg/daemon/ceph/osd/volume_test.go

```diff
@@ -24,11 +24,32 @@ import (
 
 	"github.com/pkg/errors"
 	"github.com/rook/rook/pkg/clusterd"
+	cephconfig "github.com/rook/rook/pkg/daemon/ceph/config"
+	oposd "github.com/rook/rook/pkg/operator/ceph/cluster/osd"
+	cephver "github.com/rook/rook/pkg/operator/ceph/version"
 	exectest "github.com/rook/rook/pkg/util/exec/test"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+var initializeBlockPVCTestResult = `{
+    [   {
+            "Physical volume : /dev/sdb",
+            "Volume group : ceph-125c7a9a-a9aa-450f-98da-28a4ac5945bc",
```

Comment:
quotes are not used correctly here and in some other places in the output as well.